packages/volto/src/components/manage/Blocks/Block/__snapshots__/BlocksForm.test.jsx.snap

```diff
@@ -114,5 +114,152 @@ exports[`Allow override of blocksConfig 1`] = `
+            aria-roledescription="sortable"
+            class="action drag"
+            data-cypress="draggable-handle"
+            role="button"
+            tabindex="0"
+          >
+            <svg
+              class="icon"
+              style="width: auto; fill: currentColor;"
+              viewBox=""
+              xmlns=""
+            />
+          </button>
+          <span
+            class="text"
+          >
+             
+          </span>
+          <button
+            class="action delete"
+            tabindex="0"
+          >
+            <svg
+              class="icon"
+              style="width: auto; fill: currentColor;"
+              viewBox=""
+              xmlns=""
+            />
+          </button>
+        </div>
+      </li>
+      <li
+        class="tree-item-wrapper"
+        difflib="[object Object]"
+        dndkitcore="[object Object]"
+        moment="[object Object]"
+        prettierparserhtml="[object Object]"
+        prettierstandalone="[object Object]"
+        prismcore="[object Object]"
+        reactbeautifuldnd="[object Object]"
+        reactdates="[object Object]"
+        reactdnd="[object Object]"
+        reactdndhtml5backend="[object Object]"
+        reactselect="[object Object]"
+        reactselectasync="[object Object]"
+        reactselectasynccreateable="[object Object]"
+        reactselectasyncpaginate="[object Object]"
+        reactselectcreateable="[object Object]"
+        reactsortablehoc="[object Object]"
+        reactvirtualized="[object Object]"
+        role="presentation"
+        rrule="[object Module]"
+        style="--spacing: 0px;"
+        toastify="[object Object]"
+      >
+        <div
+          class="tree-item depth-0"
+          style="transition: transform 200ms ease;"
+        >
+          <button
+            aria-describedby="DndDescribedBy-0"
+            aria-disabled="false"
+            aria-roledescription="sortable"
+            class="action drag"
+            data-cypress="draggable-handle"
+            role="button"
+            tabindex="0"
+          >
+            <svg
+              class="icon"
+              style="width: auto; fill: currentColor;"
+              viewBox=""
+              xmlns=""
+            />
+          </button>
+          <span
+            class="text"
+          >
+             
+          </span>
+          <button
+            class="action delete"
+            tabindex="0"
+          >
+            <svg
+              class="icon"
+              style="width: auto; fill: currentColor;"
+              viewBox=""
+              xmlns=""
+            />
+          </button>
+        </div>
+      </li>
+      <div
+        id="DndDescribedBy-0"
+        style="display: none;"
+      >
+        
+    To pick up a draggable item, press the space bar.
+    While dragging, use the arrow keys to move the item.
+    Press space again to drop the item in its new position, or press escape to cancel.
```

Comment:
I think this should be translatable.

Reply:
nm, this is a test, not a core UI thing, but the message can be customized:

https://docs.dndkit.com/guides/accessibility#screen-reader-instructions